packages/components/src/elevation/README.md

```diff
@@ -63,3 +63,15 @@ Dimensional offsets (margin) for the shadow.
 
 Size of the shadow, based on the Style system's elevation system. The `value` determines the strength of the shadow, which sense of depth.
 In the example below, `isInteractive` is activated to give a better sense of depth.
+
+```jsx
+import { __experimentalElevation as Elevation } from '@wordpress/components';
```

Comment:
Nit: We should probably change the `import` statement in the other example in this README (line 10), so that it imports `Elevation` by renaming `__experimentalElevation`